pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/proc_previewing-modules-in-pantheon.adoc

```diff
@@ -1,21 +1,22 @@
 [id="previewing-modules-in-pantheon_{context}"]
-= Previewing modules in {ProductShortName}
+= Previewing {ContentTerm} in {ProductShortName}
 
-Preview a module that has been imported in {ProductShortName}. 
+[role="_abstract"]
+Preview {ContentTerm} that has been imported in {ProductShortName}. 
 
 .Prerequisites
 * You are logged in.
 +
-If you are not logged in, you can still see the rendered module if it is already published to the customer portal.
+If you are not logged in, you can still see the rendered {ContentTerm} if it is already published to the {PublishingPortal}.
 
-* The module to be previewed has been imported in {ProductShortName}.
+* The {ContentTerm} to be previewed has been imported in {ProductShortName}.
 
 .Procedure
 . Navigate to the link:{LinkToSearchPage}[Search page].
-. Click the name of the module you want to preview.
+. Click the name of the {ContentTerm} you want to preview.
 +
-If you do not see the module listed, search for the modules using the search box.
+If you do not see the {ContentTerm} listed, search for the {ContentTerm} using the search box.
```

Comment:
The meaning changes very slightly here (from "modules" to "module"). Do we have an attribute for `{ContentTermPlural}` or something similar?

Reply:
Hi @robin-owen, currently :ContentTerm: is replaced with content.

Reply:
suggestion:
Change the second *{ContentTerm}* to *it*.

Reply:
I did not understand, do you mean "If you do not see the {ContentTerm} listed, search for the content using the search box."?

Also, the line above reads a bit funny - "Click the name of the content you want to preview." Any suggestions? I believe we use generic "content" to mean either an assembly or a module, but this is one instance it doesn't work. The line will read much better as "Click the name of the module or assembly you want to preview". Wdyt?

Reply:
done